with_items.yml

```diff
@@ -0,0 +1,27 @@
+# This example workflow demonstrates how to run the same task for each item in a collection.
+#
+# Workflows in the Civis Platform are written in YAML and a workflow DSL
+# (domain specific language) called Mistral.
+#
+# See this website, https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html,
+# for an introduction to YAML.
+#
+# See the Mistral documentation, https://docs.openstack.org/mistral/pike/user/dsl_v2.html,
+# for a description of the Mistral DSL.
+
+version: '2.0'  # you always need this key to specify version 2 of the mistral DSL
+workflow:
+  input:
+    # The collection referenced with YAQL should be under input
```

Comment:
Does it have to be?  I think you can also specify the list inline, without making it an input parameter that you can modify at runtime.  But an example like this, where it's modifiable at runtime, is also useful.

Reply:
I haven't been able to get with-items to work without putting it in input. There's probably a more complicated way to reference things specified elsewhere but I don't know what it is.